bin/buttercup

```diff
@@ -31,6 +31,11 @@ Buttercup options:
                           which case tests will be run if they match
                           any of the given patterns.
 
+--silent-skipping, -s   When a test is skipped due to patterns or
+                          `xit' form, don't print anything about it.
```

Comment:
That quoting style wont work in this type of heredoc. You need to switch to ""

Reply:
Fixed.